src/ExtendedView.js

```diff
@@ -5,14 +5,26 @@ function ExtendedView(props) {
   console.log(singleMovie)
 
   return(
-    <div className="extended-view">
-      <img
-        className="expanded-movie"
-        key={singleMovie.id}
-        src={singleMovie.backdrop_path}
-      />
-      <button onClick={() => props.changeExtendedState()}>BACK</button>
-    </div>
+    <div className="extended-page">
+      <button className="back" onClick={() => props.changeExtendedState()}>BACK</button>
+        <div className="movie-poster">
+          <div
+            className="movie-image"
+            style={{backgroundImage: `url(${singleMovie.backdrop_path})`, backgroundSize: "cover", height: "75vh"}}>
```

Comment:
We might want to drop down each new property in the styling rule object to a new line, like it would be done in a CSS file. This could just enhance our code readability :)

Big fan of the use of inline styling here!